src/framework/theme/components/tabset/tabset.component.ts

```diff
@@ -37,13 +44,15 @@ import { NbIconConfig } from '../icon/icon.component';
 @Component({
   selector: 'nb-tab',
   template: `
-    <ng-container *ngIf="init">
+    <ng-template portalHost #container>
       <ng-content></ng-content>
```

Comment:
Check if components projected into tabs are actually not created.

Reply:
done